packages/mds-config-service/client.ts

```diff
@@ -0,0 +1,15 @@
+import fs from 'fs'
+import { promisify } from 'util'
+import { NotFoundError } from '@mds-core/mds-utils'
+
+export const getSettings = async <TSettings extends object>(name = 'settings'): Promise<TSettings> => {
+  const { MDS_CONFIG_PATH = '.' } = process.env
+  const path = MDS_CONFIG_PATH.endsWith('/') ? MDS_CONFIG_PATH : `${MDS_CONFIG_PATH}/`
```

Comment:
Should it `|| '\'` for Windows locations or does it not matter?

Reply:
Node.js `path` modules handles all that.